CHANGELOG

```diff
@@ -1,11 +1,12 @@
 v3.4.5 (XXXX-XX-XX)
 -------------------
 
-* when restarting a follower in active failover mode, try an incremental sync instead
-  of a full resync
+* fixed internal issue #3757 : when restarting a follower in active failover mode,
+  try an incremental sync instead of a full resync. Fixed also the case where a
+  double resync was made
 
-* add "PRUNE <condition>" to AQL Traversals. This allows to early abort searching of
-  unnecessary branches within a traversal.
+* add "PRUNE <condition>" to AQL Traversals (internal issue #3068). This allows to early
```

Comment:
We could also link this public issue: https://github.com/arangodb/arangodb/issues/8112
It refers to another, older issue, but the title is pretty negative...

Reply:
Let's keep it to internal